recipes/glad/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "glad" %}
+{% set version = "0.1.36" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Dav1dde/glad/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 8470ed1b0e9fbe88e10c34770505c8a1dc8ccb78cadcf673331aaf5224f963d2
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x.x.x') }}
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('cxx') }}
```

Comment:
```suggestion
```
The logs indicate that the cxx compiler may not be used. OR we need to add ignore_run_export_from the cxx compiler to the build section because the cxx standard library is unused.

Reply:
Good catch! The library is indeed a C-only library, and this is correctly declared in the CMake level (see https://github.com/Dav1dde/glad/blob/v0.1.36/CMakeLists.txt#L22), so we can just drop the dependency.